app/src/main/java/org/fossasia/openevent/fragments/BookmarksFragment.java

```diff
@@ -29,11 +29,17 @@
  * Created by manan on 22-05-2015.
  */
 public class BookmarksFragment extends Fragment {
-    SessionsListAdapter sessionsListAdapter;
+    static SessionsListAdapter sessionsListAdapter;
```

Comment:
nope this isn't right, firstly, `SessionDetailActivity` should not access the Bookmarks fragment plus, you shouldn't need to access cached objects like this.